.buildkite/pipeline.yml

```diff
@@ -81,20 +108,20 @@ steps:
         command:
           - --app=build/output.ipa
           - --farm=bs
-          - --device=IOS_16
+          - --device=IOS_18
           - --a11y-locator
           - --fail-fast
           - --retry=2
           - --order=random
       test-collector#v1.10.2:
         files: "reports/TEST-*.xml"
         format: "junit"
-        branch: "^v51$"
+        branch: "^v52$"
     concurrency: 5
     concurrency_group: 'browserstack-app'
     concurrency_method: eager
 
-  - label: ':runner: expo iOS 14'
+  - label: ':runner: expo iOS 16'
```

Comment:
Minimum supported version for Expo 52 is `15.1` but browserstack capabilities do not appear to match up, so we'll up(down?)date this later.